pkg/test/framework/components/galley/native.go

```diff
@@ -280,6 +280,7 @@ func (c *nativeComponent) restart() error {
 	a.ExcludedResourceKinds = nil
 	a.EnableServiceDiscovery = true
 	a.ValidationArgs.EnableValidation = false
+	a.ValidationArgs.DisableReconcileWebhookConfiguration = true
```

Comment:
Why is this disabled for the integration test?

Reply:
I cannot recall the specific error for the integration test. I think that should be the same reason we disable validation.